src/Scratch/Install-Sitecore12.ps1

```diff
@@ -0,0 +1,78 @@
+<#
```

Comment:
Needs a better file name and an updated doc header. Maybe Install-SitecoreDevInstance?